docs/docfx.json

```diff
@@ -1274,46 +1276,90 @@
       "src": "breadcrumb",
       "dest": "office_js_breadcrumb"
     },
-    
+	
```

Comment:
Minor nit, but your editor seems to add tabs instead of spaces. I only bring this up because some keyboard shortcut sample broke because of it.